core/src/main/java/mx/yellowme/youst/core/utils/Utils.kt

```diff
@@ -35,14 +39,25 @@ fun Activity.readJson(named: String): String? {
     return classLoader?.readJson(named)
 }
 
-inline fun <reified T : Any> Activity.loadJsonArray(named: String) : List<T>? {
+inline fun <reified T : Any> Activity.loadJsonArray(named: String): List<T>? {
     return readJson(named)?.asJsonArrayOf()
 }
 
-inline fun <reified T : Any> Activity.loadJsonObject(named: String) : T? {
+inline fun <reified T : Any> Activity.loadJsonObject(named: String): T? {
     return readJson(named)?.asJsonObject()
 }
 
+fun Activity.composeEmail(address: String, subject: String) {
```

Comment:
Since this method also launches the intent I think we should rename it to be something similar to:

`lanchExternalEmailApp`, `lanchExternalEmailChooser`